fullykiosk/__init__.py

```diff
@@ -47,91 +52,136 @@ def deviceInfo(self):
     def settings(self):
         return self._settings
 
-    async def startScreensaver(self):
-        await self.sendCommand("startScreensaver")
+    # Configurations
 
-    async def stopScreensaver(self):
-        await self.sendCommand("stopScreensaver")
+    async def setConfigurationString(self, setting, stringValue):
+        await self.sendCommand("setStringSetting", key=setting, value=stringValue)
+
+    async def setConfigurationBool(self, setting, boolValue):
+        await self.sendCommand("setBooleanSetting", key=setting, value=boolValue)
+
+    # Screen, screensaver
 
     async def screenOn(self):
         await self.sendCommand("screenOn")
 
     async def screenOff(self):
         await self.sendCommand("screenOff")
 
-    async def setScreenBrightness(self, brightness):
-        await self.sendCommand(
-            "setStringSetting", key="screenBrightness", value=brightness
-        )
+    async def forceSleep(self):
+        await self.sendCommand("forceSleep")
 
-    async def setAudioVolume(self, volume, stream=None):
-        await self.sendCommand("setAudioVolume", level=volume, stream=stream)
+    async def startScreensaver(self):
+        await self.sendCommand("startScreensaver")
 
-    async def restartApp(self):
-        await self.sendCommand("restartApp")
+    async def stopScreensaver(self):
+        await self.sendCommand("stopScreensaver")
 
-    async def loadStartUrl(self):
-        await self.sendCommand("loadStartUrl")
+    async def startDaydream(self):
+        await self.sendCommand("startScreensaver")
```

Comment:
Shouldn't this send the `startDaydream` command instead of `startScreensaver`?
Same for stopDaydream below

Reply:
Yes, I copypasted the lines, and forgot to change that part as well :)